app/javascript/src/contexts/sideMenuStatus.js

```diff
@@ -0,0 +1,4 @@
+import { createContext } from "react";
+
+const SideMenuStatusContext = createContext();
+export default SideMenuStatusContext;
```

Comment:
Do we really need this?

Reply:
Why do we need the state to be persistent across pages? We don't need to persist the state of collapsible menu across pages.